feat(search-form): migrate component search form - FRONT-1775 (#1760)
ec-europa/europa-component-library: src/implementations/twig/components/search-form/README.md

# ECL Search Form

npm package: `@ecl/twig-component-search-form`

```shell
npm install --save @ecl/twig-component-search-form
```

### Parameters

> - **"text_input"** (associative array) default: A predefined structure for EC Text Input

Reviewer: ```suggestion
- **"text_input"** (associative array) default: Predefined structure for the text-Input component
```

> - **"button"** (associative array) default: A predefined structure for EC Button

Reviewer: ```suggestion
- **"button"** (associative array) default: Predefined structure for the Button component
```

- **"extra_form_elements"** (optional) (string) (default: '') Extra elements for the form
- **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
- **"extra_attributes"** (optional) (array) (default: []) Extra attributes
  - "name" (string) Attribute name, eg. 'data-test'
  - "value" (string) Attribute value, eg: 'data-test-1'

> - **"\_compliance"** (boolean) (default: false) Activates debug

Reviewer: ```suggestion
- **"\_compliance\_"** (boolean) (default: false) Activates debug
```

> - **"_compliance_inner_check_"** (boolean) (default: false) Inline compliance report

Reviewer: ```suggestion
- **"\_compliance_inner_check\_"** (boolean) (default: false) Inline compliance report
```
